### PR TITLE
README: Add x64.

### PR DESCRIPTION
**Pull request by GamingCrafthd**
Since Serenity supports x64 now, I made a PR to update the README accordingly.

### CODE REVIEW

**Review by yegor-github — APPROVED** [review]

**Comment by ADKaster** [comment]
You'll need to amend your commit and force push it with a fixed title (and hopefully description) In order for the CI linter to approve of the change.

something like:
```
git commit --amend
< change commit tile to something like : Meta: Add x86_64 to supported architectures in README >
git push -f my-remote
```

Note that you can't do this from the GitHub website UI.

**Comment by awesomekling** [comment]
Please use x86_64 consistently instead of mixing x64 and x86_64

**Comment by awesomekling** [comment]
Please observe the [contributing guidelines](https://github.com/SerenityOS/serenity/blob/3dd40535c1488e6437923fa5e90f73d3eeb9e108/CONTRIBUTING.md#code-submission-policy), specifically:
> Squash your commits when making revisions after a patch review.

File: README.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 # SerenityOS
 
-Graphical Unix-like operating system for x86 computers.
+Graphical Unix-like operating system for x86 and x86_64 computers.
```
Comment (awesomekling) [inline README.md:3]: This change is not necessary. "x86 computers" includes both 32-bit and 64-bit CPUs.

**Review by awesomekling — CHANGES_REQUESTED** [review]
When squashing commits, please edit the resulting commit message so it looks good and doesn't repeat itself.

There is no reason to keep both of the commit messages after doing a squash.

**Pull request merged**